src/graphql-schema/resolvers.js

```diff
@@ -0,0 +1,45 @@
+// const ssbResolvers = require('./ssb')
+
+module.exports = function Resolvers (sbot) {
+  // const {
+  //   getProfile,
+  //   gettersWithCache
+  // } = ssbResolvers(sbot)
+
+  function getProfile (opts) {
+    return {
+      id: opts.id,
+      name: 'Cherese',
+      image: '',
+      // publicWebHosting
+
+      threads: [],
+      // threads (started: Boolean): [Thread]
+      // threadCount: Int
+
+      followers: [],
+      followersCount: 0,
+
+      following: [],
+      followingCount: 0
+    }
+  }
+
+  const resolvers = {
+    Query: {
+      getProfile: (_, opts) => getProfile(opts)
+
+      // TODO
+      // getThread(key: ID!, preview: Boolean): Thread
+      // getSample(limit: Int): [Thread]
+      // getProfiles(limit: Int) [Profile]
+    }
+    // Mutation: {
+    // }
+  }
+
+  return {
+    resolvers
+    // gettersWithCache
+  }
```

Comment:
I would call these files:
- src/graphql/typeDefs.js
- src/graphql/resolvers.js
- src/graphql/index.js - the server itself

I don't really know what 'graphql-schema' is?